instrumentation/pekko/pekko-http-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/pekkohttp/v1_0/server/tapir/RouteWrapper.java

```diff
@@ -29,14 +34,22 @@ public RouteWrapper(
   }
 
   public class Finalizer implements PartialFunction<Try<RouteResult>, Unit> {
```

Comment:
could this class be package private?

Reply:
I can change that, though it wasn't introduced in this PR.